src/aiida/cmdline/params/arguments/main.py

```diff
@@ -71,7 +71,7 @@
 
 PROCESS = OverridableArgument('process', type=types.ProcessParamType())
 
-PROCESSES = OverridableArgument('processes', nargs=-1, type=types.ProcessParamType())
+PROCESSES = OverridableArgument('processes', nargs=-1, type=types.ProcessParamType(), metavar='[PROCESSES]...')
```

Comment:
This is already the autogenerated metavar
```suggestion
PROCESSES = OverridableArgument('processes', nargs=-1, type=types.ProcessParamType())
```

Reply:
done